test/integration/preferences.test.js

```diff
@@ -2,45 +2,44 @@ import { expect } from 'chai';
 import Patch from '../../dist/index';
 const patch = Patch(process.env.SANDBOX_API_KEY);
 
+async function getProjectId() {
+  const retrieveProjectResponse = await patch.projects.retrieveProjects();
+  return retrieveProjectResponse.data[0].id;
+}
+
 describe('Preferences Integration', async function () {
```

Comment:
@kleinjm I bit the bullet and rewrote this test in the same style as the Ruby one.